weaviate/collection/classes.py

```diff
@@ -547,6 +547,26 @@ def to_dict(self) -> Dict[str, Any]:
         return ret_dict
 
 
+class IncludesModel(BaseModel):
+    def to_include(self) -> str:
+        include: List[str] = []
+        for field, value in self:
+            if value:
+                include.append(field)
+        return ",".join(include)
+
+
+class GetObjectByIdIncludes(IncludesModel):
+    classification: bool = False
+    vector: bool = False
+
+
+class GetObjectsIncludes(IncludesModel):
+    classification: bool = False
+    featureProjection: bool = Field(False, alias="feature_projection")
```

Comment:
I'm pretty sure that classification and feature projection are not yet included in GRPC - what happens if you set them to true? I assume that it is just ignored?

Reply:
These includes options are used when sending GET requests to `/v1/objects` so don't touch the gRPC. Though, this seems to be highlighting a potential naming convention conflict/confusion 🤔

Reply:
ah right

yeah that is a bit unfortunate that the have different default options on what they send back.

Reply:
let's discuss directly, this PR doesn't make it more confusing than before :)